src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2010.ProvideCodeSnippets_WithEmbeddedContent.sarif

```diff
@@ -0,0 +1,69 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "CodeScanner",
+          "version": "1.0"
+        }
+      },
+      "originalUriBaseIds": {
+        "TEST_ROOT": {
+          "uri": "file://c:/test/"
+        }
+      },
+      "artifacts": [
+        {
+          "contents": {
+            "text": "File Source"
+          },
+          "location": {
+            "uri": "src/test.c"
+          }
+        },
+        {
+          "contents": {
+            "text": "File Source"
+          },
+          "location": {
+            "uri": "test2.c",
+            "uriBaseId": "TEST_ROOT"
+          }
+        }
+      ],
+      "results": [
+        {
+          "ruleId": "TST0001",
+          "level": "error",
+          "message": {
+            "text": "Some testing occurred."
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  // this test will NOT generate a valid AbsoluteUri, but it will find in artifacts
+                  // So, it don't need a snippet
```

Comment:
Comments not valid in a SARIF file. You can put a comment in the property bag, for example:
```
"physicalLocation": {
  "artifactLocation": {
    "uri": "src/test.c"
  },
  "properties": {
    "comment": "This test will NOT generate a valid AbsoluteUri, but it will find find the file in the artifacts array. So it doesn't need a snippet."
  }
}
```